packages/contrast-colors/index.js

```diff
@@ -488,27 +785,46 @@ function luminance(r, g, b) {
 }
 
 function contrast(color, base, baseV) {
+  if(baseV == undefined) { // If base is an array and baseV undefined
+    let colorString = String(`rgb(${base[0]}, ${base[1]}, ${base[2]})`);
+    
+    baseLightness = Number((d3.hsluv(colorString).v));
+    if(baseLightness > 0) {
+      baseV = Number((baseLightness / 100).toFixed(2));
```

Comment:
Are you trying to round this? Converting it to a string and then back to a number seems weird. Generally, we use `Math.round()` and multiply things. However, this article is interesting: https://www.jacklmoore.com/notes/rounding-in-javascript/